File: solutions/ruby/raindrops/1/raindrops.rb
module Raindrops
  def self.convert(n)
    result = ""
    result << "Pling" if n % 3 == 0
    result << "Plang" if n % 5 == 0
    result << "Plong" if n % 7 == 0
    return result unless result.empty?
    n.to_s
  end
end

File: solutions/python/raindrops/1/raindrops.py
def convert(number):
    def div(x):
        return number % x == 0
    result = ("Pling" if div(3) else "") \
        + ("Plang" if div(5) else "") \
        + ("Plong" if div(7) else "")
    if result:
        return result
    return str(number)
